src/js/components/inspectors/Area.tsx

```diff
@@ -1,39 +1,68 @@
 'use strict';
 const INTERPOLATE = require('../../constants/interpolate');
+const getInVis = require('../../util/immutable-utils').getInVis;
 
 import * as React from 'react';
 import {connect} from 'react-redux';
 import {PrimType} from '../../constants/primTypes';
 import {Property} from './Property';
+import {ExtentProperty} from './ExtentProperty';
+import {State} from '../../store';
 
-interface AreaProps {
+const Area = {
+  'ORIENT': [
+    'vertical',
+    'horizontal'
+]};
+
+interface OwnProps {
   primId: number,
   primType: PrimType
 }
 
-class BaseArea extends React.Component<AreaProps> {
+interface StateProps {
+  orient: string;
+}
+
+function mapStateToProps(reduxState: State, ownProps: OwnProps): StateProps {
+  const primId = ownProps.primId;
+  return {
+    orient: getInVis(reduxState, `signals.lyra_area_${primId}_orient.value`)
```

Comment:
i should have said something about this, but we're trying to move away from the `getInVis` helper function in favor of this syntax:

```
reduxState.getIn(['vis', 'present', 'signals', `lyra_area_${primId}_orient`, 'value'])
```

and i think it would be even better not to hardcode the signal name, just in case something about that changes down the line. so:

```
const orientSignal = reduxState.getIn(['vis', 'present', 'marks', str(primId), 'encode', 'update', 'orient', 'signal']);
return {
  orient: reduxState.getIn(['vis', 'present', 'signals', orientSignal, 'value'])
}
```

and if this fails, it'll be easier to tell right away what's going wrong.